src/react/experimental/viewport-transition-utils.js

```diff
@@ -4,30 +4,46 @@ import {projectFlat, unprojectFlat} from 'viewport-mercator-project';
 import {Vector2} from 'math.gl';
 
 const EPSILON = 0.01;
+export const VIEWPORT_PROPS = ['longitude', 'latitude', 'zoom', 'bearing', 'pitch',
```

Comment:
Why interpolate `minZoom` and `maxZoom`? It may cause weird zoom output during interpolation.
If it's deemed necessary, should we add `minPitch` and `maxPitch` too?